_includes/setting.html

```diff
@@ -7,6 +7,8 @@ <h3 id="{{ setting.name | slugify }}">
   {% endif %}
 </h3>
 
+{% if is_singer == false %}
```

Comment:
Is `is_singer` actually available here? As far as I know, variables have to be passed into included partials explicitly and then referenced as `include.<var>` as in line 1 of this file.

Reply:
If you compare the settings on https://deploy-preview-541--meltano-hub.netlify.app/extractors/auth0 and https://deploy-preview-541--meltano-hub.netlify.app/taps/auth0 it does seem to work. 🤷